### DISCUSSION TITLE
Changing Preprocessor Resolution

### DISCUSSION THREAD

**Discussion by OrganicBeej** [Mikubill/sd-webui-controlnet#2084] [Q&A] [answered]
Hi :),

I see now that I can't set the Preprocessor value to be what I want? It's defaulting to -1 but for my workflow (Im now in A1111 on iMac M1 v1.6) has always been (previously) to set smaller image size but force the Preprocessor to be larger as this really helps with render quality.

Is there anywhere in the code where I can edit this so it works like it used too please? I was able to set a 512 image (for 1.5 models) but have my DepthMap render-out first at 768 (or larger), drop this into the CN img window and then set Preprocessor to 'none'.
This would give me a larger depthmap but keep the img size smaller and I can't achieve it now, not without matching the image size to the larger depthmap and that's not what I want.

Also, I used to be able to drag the rendered, pre-processed image from right window to left but this doesn't work any more?

Would appreciate some advice, thnks :)

**Comment by lllyasviel** [comment, marked as answer]
uncheck "pixel-perfect"

**Reply by OrganicBeej** [reply]
I feel stoopid now ;)
Thnks
